inspirehep/modules/records/wrappers.py

```diff
@@ -123,18 +125,22 @@ def publication_information(self):
         """
         pub_info_list = []
         for pub_info in self['publication_info']:
-            if pub_info.get('journal_title', '') or pub_info.get('pubinfo_freetext', ''):
-                pub_info_list.append({
-                    'journal_title': pub_info.get('journal_title', ''),
-                    'journal_volume': pub_info.get('journal_volume', ''),
+            item = {}
+            if pub_info.get('journal_title') or pub_info.get('pubinfo_freetext'):
+                item.update({
+                    'journal_title': pub_info.get('journal_title'),
+                    'journal_volume': pub_info.get('journal_volume'),
                     'year': str(pub_info.get('year', '')),
-                    'journal_issue': pub_info.get('journal_issue', ''),
+                    'journal_issue': pub_info.get('journal_issue'),
                     'page_start': str(pub_info.get('page_start', '')),
                     'page_end': str(pub_info.get('page_end', '')),
-                    'artid': pub_info.get('artid', ''),
-                    'pubinfo_freetext': pub_info.get('pubinfo_freetext', '')
+                    'artid': pub_info.get('artid'),
+                    'pubinfo_freetext': pub_info.get('pubinfo_freetext'),
                 })
 
+            if item:
```

Comment:
@drjova ok I had to add this check also because otherwise empty objects end up in the `pub_info_list`...